### ISSUE TITLE
Windows doesn't see Leonardo as joystick after uploading N64PadToUSB

### ISSUE THREAD

**Issue by Jesright73** [SukkoPera/N64PadForArduino#2]
Uploaded N64PadToUSB to Leonardo with required libraries. Connected controller to appropriate pins (3.3v, data3, Ground), 1k resistor for pull-up from 3.3 to data 3. 
After upload, Windows detects Leonardo as an _Unspecified_ device, not as a joystick/gamepad.

OS: Windows 10 Home
Arduino Software: 1.8.15
Arduino Leonardo MEGA32U4

**Comment by SukkoPera** [comment]
I haven't been using Windows in almost 20 years, can't say much, sorry. Maybe try the Arduino joystick Library examples first: https://github.com/MHeironimus/ArduinoJoystickLibrary

**Comment by capitaineflam25** [comment]
You'll have to change Joystick definition from **JOYSTICK_TYPE_MULTI_AXIS** to **JOYSTICK_TYPE_JOYSTICK** (Line 84 of [N64PadToUSB.ino](https://github.com/SukkoPera/N64PadForArduino/blob/master/examples/N64PadToUSB/N64PadToUSB.ino)) and it will be OK on Windows.

**Comment by SukkoPera** [comment]
Closing since there was no update in a long time.

**Comment by SaintBepsi** [comment]
> You'll have to change Joystick definition from **JOYSTICK_TYPE_MULTI_AXIS** to **JOYSTICK_TYPE_JOYSTICK** (Line 84 of [N64PadToUSB.ino](https://github.com/SukkoPera/N64PadForArduino/blob/master/examples/N64PadToUSB/N64PadToUSB.ino)) and it will be OK on Windows.

This legit works. Also for some reason Dpad and stick vere functioning as one in N64PadToUSB, so had to edit that too.

**Comment by SukkoPera** [comment]
Can you please provide a Pull Request with the required changes?

**Comment by SaintBepsi** [comment]
> Can you please provide a Pull Request with the required changes?

I'm gonna be honest, I don't even know what a pull request is. I'll just attach the code that I ended up uploading to my board.
I removed things I did not need and changed joystick type. I'm currently trying to figure out how to make DPAD act as hat switch instead of X/Y axis, cuz I encountered some minor inconviniences in emulators. 
[new 1.txt](https://github.com/user-attachments/files/18994195/new.1.txt)
I don't see API commands in joystick library that I need to do HAT switch thing, so I might just turn dpad into independant buttons
